CMakeLists.txt

```diff
@@ -385,8 +386,32 @@ message(STATUS "All defines at startup: ${COMPILE_DEFINES}")
 # -----------------------------------------------------------------------------
 # Setup library build path (this is **not** the library install path)
 # -----------------------------------------------------------------------------
-set(CMAKE_LIBRARY_OUTPUT_DIRECTORY ${PROJECT_BINARY_DIR}/lib)
-set(CMAKE_ARCHIVE_OUTPUT_DIRECTORY ${PROJECT_BINARY_DIR}/lib)
+if(NOT WIN32)
```

Comment:
You're branch condition is wrong. This is not windows specific. It depends on whether or not the CMake generator is single or multi-configuration. For example it is perfectly possible to use Ninja on Windows which is a single configuration generator. Instead your condition should be.

```
if (DEFINED CMAKE_CONFIGURATION_TYPES)
  # Multi-configuration generator (e.g. XCode, Visual Studio)
else()
  # Single configuration (e.g. Ninja, Unix Makefiles)
endif()
```